pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -746,6 +746,8 @@ type Artifact struct {
 type PodGC struct {
 	// Strategy is the strategy to use. One of "OnPodCompletion", "OnPodSuccess", "OnWorkflowCompletion", "OnWorkflowSuccess"
 	Strategy PodGCStrategy `json:"strategy,omitempty" protobuf:"bytes,1,opt,name=strategy,casttype=PodGCStrategy"`
+	// LabelSelector is the label selector to select completed pods that match the labels to be added to the pod GC queue.
+	LabelSelector string `json:"labelSelector,omitempty" protobuf:"bytes,2,opt,name=labelSelector"`
```

Comment:
I think this should probably be `metav1.LabelSelector`

Reply:
Good point. Updated.